packages/ui/src/components/Typography/Typography.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { HStack } from '../Stack';
+import { IsAnchor, Typography } from './index';
```

Comment:
Import from the `./Typografy.tsx` file as it's an easier way to protect the app from dependency cycles.